packages/react-experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -59,11 +60,28 @@ const _SelectedItemsList = <TItem extends BaseSelectedItem>(
     [items],
   );
 
+  const _onFocus = (ev: React.FocusEvent<HTMLDivElement>) => {
+    // If we have an input, set focus to it so we can pick up the copy keyboard command
+    hiddenInput.current?.focus();
+  };
+
   const SelectedItem = props.onRenderItem;
   return (
     <>
       {items.length > 0 && (
-        <div role={'list'}>
+        <div role={'list'} onFocus={_onFocus}>
+          <input
+            // Add a zero zero size input. This is to pick up the copy command when we have
+            // keyboard focus in the list
+            // Focus is set to the input when the list gets focus via the _onFocus function
+            className={'ms-SelectedItemsList-copyInput'}
+            ref={hiddenInput}
+            hidden={!props.getItemCopyText}
+            style={{ height: '0px', width: '0px', border: 'none', outline: 'none' }}
```

Comment:
i know this is zero size input but is there a way we can put this in the styles file for cleanliness?

Reply:
yeah good call will update